gcp-deployment-manager/Makefile

```diff
@@ -0,0 +1,127 @@
+# Default project variables.
+include project.env
+# Project-specific overrides. These should be version controlled.
+PROJECT_OVERRIDES:=project-override.env
+# Untracked local overrides file, for development purposes.
+LOCAL_OVERRIDES:=local-override.env
+DOCKER_ENV_FILE:=--env-file=project.env
+
+ifneq ("$(wildcard $(PROJECT_OVERRIDES))","")
+    include $(PROJECT_OVERRIDES)
+	DOCKER_ENV_FILE+=--env-file=$(PROJECT_OVERRIDES)
+endif
+
+ifneq ("$(wildcard $(LOCAL_OVERRIDES))","")
+	include $(LOCAL_OVERRIDES)
+	DOCKER_ENV_FILE+=--env-file=$(LOCAL_OVERRIDES)
+endif
+
+export
+
+login:
+	docker run -ti --name gcloud-config google/cloud-sdk:$(CLOUD_SDK_IMAGE_VERSION) gcloud auth login
+
+guard-%:
+	@ if [ "${${*}}" = "" ]; then \
+		echo "Environment variable $* not set"; \
+		exit 1; \
+	fi
+
+bash:
+	docker run -it --rm \
+	--volumes-from gcloud-config \
+	-v `pwd`:/opt/deployment \
+	$(DOCKER_ENV_FILE) \
+	-w /opt/deployment \
+	google/cloud-sdk:$(CLOUD_SDK_IMAGE_VERSION) /bin/bash
+
+clean-container:
+	docker rm -v gcloud-config
+
+create:
+	docker run -it --rm \
```

Comment:
This is a great candidate to go into a constant and be used anywhere `gcloud` is used.

```
docker run -it --rm \
	--volumes-from gcloud-config \
	-v `pwd`:/opt/deployment \
	$(DOCKER_ENV_FILE) \
	google/cloud-sdk:$(CLOUD_SDK_IMAGE_VERSION) \
        gcloud
```

Reply:
Update.